feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/captive.c

```diff
@@ -228,134 +226,133 @@ void set_captive_portal_state(struct schema_Wifi_VIF_State *vstate,
 	return;
 }
 
-void vif_state_captive_portal_options_get(struct schema_Wifi_VIF_State *vstate, struct uci_section *s)
+void vif_state_captive_portal_options_get(struct schema_Wifi_VIF_State *vstate)
 {
 	int i;
 	int index = 0;
 	const char *opt;
 	char *buf = NULL;
 	struct blob_attr *tc[__NDS_ATTR_MAX] = { };
-	struct uci_element *e = NULL;
+	struct uci_section *cp_section;
 
 	uci_load(cap_uci, "opennds", &opennds);
-	uci_foreach_element(&opennds->sections, e) {
```

Comment:
This large change is really mostly white-space.  It would be much easier to review if you had replaced the "if !strcmp()) {" at line 243 with just a "{".  Then the indentation could stay the same.

I don't actually suggest you do this now.

Reply:
got it, thank you